NREL/ssc
# Sixparsolve error reporting

Update #802 to fix uninitialized variable error.

To test:

1. Create a Detailed PV / No Financing case.
2. On the module page, switch to CEC Performance Model with User Entered Specifications.
3. Change short circuit current to 25 Amps
4. Click **Calculate and plot**: "Sanity check failed" error with details should appear.
5. Click **Simulate**: "Could not solve for normalized coefficients" exec fail message should appear.
